.github/workflows/main.yml

```
@@ -0,0 +1,114 @@
+    - name: Upload Frontend Artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: frontend-build
+        path: ./client/dist/client
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+
+    steps:
+    - name: Download Backend Artifact
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)